batchglm/models/glm_nb/utils.py

```diff
@@ -76,7 +78,7 @@ def compute_scales_fun(variance, mean):
     )
 
 
-def init_par(input_data, init_location, init_scale, init_model):
+def init_par(model, init_location: str, init_scale: str) -> Tuple[np.ndarray, np.ndarray, bool, bool]:
```

Comment:
Seems like a good candidate for multi-dispatch https://github.com/mrocklin/multipledispatch on the enum type we could define....we can also just make an issue for this and I can handle it.

Reply:
open an issue please. I'm not familiar with multipledispatch

Reply:
Followup: #131